.github/workflows/examples_matrix.yml

```diff
@@ -0,0 +1,90 @@
+name: ExamplesMatrix
+
+on:
+  workflow_call:
+    inputs:
```

Comment:
I'm not sure we need an extra workflow for this.

Reply:
@fabianfett This is a matter of separation of concerns. Just like when we create 2 .swift files for two different struct.
In the reporting web view, it doesn't matter if this is a sub workflow or not.

I really don't want to bring this complexity in `pull_request.yml`.  Where would you host this script if not in a separate workflow ?

Reply:
<img width="302" alt="image" src="https://github.com/user-attachments/assets/d688f5ac-4f08-4aaa-b1f8-dc204a894fd4">